.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.6
+  - 2.1.10
```

Comment:
yes, I know :) I just don't know which Stripe API version is attached to those keys. E.g. Recipients feature may work on some older APIs, but will not work now, so I want to make sure it will test against an up-to-date API version.